src/modules/package.ts

```diff
@@ -116,10 +116,15 @@ export class PackageConfig {
     }
 
     isPackageInstalled(): boolean {
-        if (!CliFileSystem.existsSync(this.getPackageDirectoryWithVersion())) {
-            return false;
+        return CliFileSystem.existsSync(this.getPackageDirectoryWithVersion());
+    }
+
+    async isPackageValidRepo(): Promise<boolean> {
+        const isValid = await packageDownloader(this).isValidRepo(this.getPackageDirectoryWithVersion());
+        if (!isValid) {
+            console.log(`... Corrupted .git directory found for: '${this.getPackageName()}:${this.version}'`);
```

Comment:
Very technical output which should be hidden behind a verbose flag.

Reply:
done